broker.go

```diff
@@ -143,7 +145,7 @@ func (b *Broker) Close() (err error) {
 	}()
 
 	if b.conn == nil {
-		return NotConnected
+		return errors.New(NotConnected)
```

Comment:
Timeliness.  We covered the ones necessary to determine the root cause of the Fidelity issue.